src/js/components/dashboard/DashboardTable.jsx

```diff
@@ -194,13 +224,13 @@ export default class DashboardTable extends React.Component {
 
         let lastColumn = "Delete";
         if(this.props.isCertified) {
-            lastColumn = "Certified By";
+            lastColumn = ["Certified By", "Certified On"];
```

Comment:
Can we name this something more appropriate? At the very least, we should make the variable name plural since it's multiple columns now?